openwisp_monitoring/device/settings.py

```diff
@@ -58,3 +72,5 @@ def get_health_status_labels():
 MAC_VENDOR_DETECTION = getattr(
     settings, 'OPENWISP_MONITORING_MAC_VENDOR_DETECTION', True
 )
+
+update_object_notification_widget()
```

Comment:
It would be better to do this in the app.ready() method, so we are sure it's executed only once.
I would do it in openwisp-controller.